core/src/test/java/org/springframework/security/authorization/ObservationAuthorizationManagerTests.java

```diff
@@ -73,7 +73,7 @@ void setup() {
 	@Test
 	void verifyWhenDefaultsThenObserves() {
 		given(this.handler.supportsContext(any())).willReturn(true);
-		given(this.authorizationManager.check(any(), any())).willReturn(this.grant);
```

Comment:
Let's leave unit tests as-is so that we know we are still backward compatible.

Reply:
Done.